src/structures/Node.ts

```diff
@@ -58,6 +58,11 @@ function check(options: NodeOptions) {
     typeof options.retryDelay !== "number"
   )
     throw new TypeError('Node option "retryDelay" must be a number.');
+  
+  if (typeof options.region !== "undefined" && 
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=7585535)
```suggestion
  if (typeof options.region !== "undefined" && 
```